packages/snaps-cli/src/cmds/serve/serve.test.ts

```diff
@@ -54,7 +54,7 @@ describe('serve', () => {
       });
       mockServer.emit('close');
       await finishPromise;
-      expect(process.exit).toHaveBeenCalledWith(1);
+      expect(process.exitCode).toStrictEqual(1);
```

Comment:
can we also remove the process.exit spies in this file?

Reply:
They are in some other files as well, e.g. `eval.test.ts`. Should review all test files to ensure these spies are removed.